pkg/build/builder/common.go

```diff
@@ -20,7 +22,14 @@ import (
 // client facing libraries should not be using glog
 var glog = utilglog.ToFile(os.Stderr, 2)
 
-const OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+const (
+	OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
```

Comment:
opened issue to track removing this stuff: https://github.com/openshift/origin/issues/11969